docker-app/qfieldcloud/subscription/models.py

```diff
@@ -823,5 +828,5 @@ class Meta:
     account = models.OneToOneField(
         UserAccount,
         on_delete=models.CASCADE,
-        related_name="current_subscription",
+        related_name="current_subscription_vw",
```

Comment:
I added the `_vw` suffix, or rename it to `_ro`, so we know it is read-only when using the ORM?

Reply:
Ah yes that would also work for me